### DISCUSSION TITLE
Could not publish deployment.

### DISCUSSION THREAD

**Discussion by vkn** [jreleaser/jreleaser#1658] [General]
Hello,

my first attempt to publish on maven central.
Github action workflow failed with the error below but the artifact was actually published. Is it possible to avoid failing and wait longer until deployment is published?

```Failed to execute goal org.jreleaser:jreleaser-maven-plugin:1.12.0:deploy (default-cli) on project profiling-unit: Execution default-cli of goal org.jreleaser:jreleaser-maven-plugin:1.12.0:deploy failed: Unexpected error: Could not publish deployment for ca708a92-e3a1-49d7-86c6-0be004b6a8cc: deployment ca708a92-e3a1-49d7-86c6-0be004b6a8cc is still transitioning```

**Comment by aalmiray** [comment]
Yes, you can modify the retry delay and max retries values, set to 20s and 60 times by default. The guide shows the names of the properties you must set

Please review the info of the specific deployer at https://jreleaser.org/guide/latest/reference/deploy/maven/index.html

**Reply by vkn** [reply]
Thanks! Looks like default is 10s x 60 times. Hope this is correct:

```xml
                    <plugin>
                        <groupId>org.jreleaser</groupId>
                        <artifactId>jreleaser-maven-plugin</artifactId>
                        <configuration>
                            <jreleaser>
                                <signing>
                                    <active>ALWAYS</active>
                                    <armored>true</armored>
                                </signing>
                                <deploy>
                                    <maven>
                                        <mavenCentral>
                                            <sonatype>
                                                <active>ALWAYS</active>
                                                <url>https://central.sonatype.com/api/v1/publisher</url>
                                                <stagingRepositories>target/staging-deploy</stagingRepositories>
                                                <retryDelay>60</retryDelay>
                                                <maxRetries>60</maxRetries>
                                            </sonatype>
                                        </mavenCentral>
                                    </maven>
                                </deploy>
                            </jreleaser>
                        </configuration>
                    </plugin>
```
